src/toolbox.ts

```diff
@@ -1483,131 +1603,163 @@ export class AnnotationResizeItem extends ToolboxItem {
     }
 
-    private save_local_storage_color(class_id: number | string, color: string): void {
-        set_local_storage_item(`RecolorActiveItem-${class_id}`, color)
+    private save_local_storage_color(
+        class_id: number | string,
+        color: string,
+    ): void {
+        set_local_storage_item(`RecolorActiveItem-${class_id}`, color);
     }
 
     private save_local_storage_gradient(gradient_status: boolean): void {
-        set_local_storage_item("RecolorActiveItem-Gradient", gradient_status)
+        set_local_storage_item("RecolorActiveItem-Gradient", gradient_status);
     }
 
     private read_local_storage(): void {
         // Loop through every valid id and see if a color has been saved for it in local storage
         for (const class_id of this.ulabel.valid_class_ids) {
-
             // Get the color from local storage based on the current class id
-            const color = get_local_storage_item(`RecolorActiveItem-${class_id}`)
+            const color = get_local_storage_item(
+                `RecolorActiveItem-${class_id}`,
+            );
 
             // Update the color if its not null
             // Additionally no need to save the color to local storage since we got it from reading local storage
-            if (color !== null) this.update_color(class_id, color, false)
+            if (color !== null) this.update_color(class_id, color, false);
         }
 
         // Then read whether or not the gradient should be on by default
-        this.gradient_turned_on = get_local_storage_item("RecolorActiveItem-Gradient")
+        this.gradient_turned_on = get_local_storage_item(
+            "RecolorActiveItem-Gradient",
+        );
     }
 
     private replace_color_pie(): void {
         // Only the current subtask's color can be changed, so only the current subtask needs to be updated
-        const current_subtask_key: string = this.ulabel.state.current_subtask
-        const current_subtask: ULabelSubtask = this.ulabel.subtasks[current_subtask_key]
+        const current_subtask_key: string = this.ulabel.state.current_subtask;
+        const current_subtask: ULabelSubtask =
+            this.ulabel.subtasks[current_subtask_key];
 
         // Get the back and front id dialog's ids
-        const id_dialog_id: string = current_subtask.state.idd_id
-        const front_id_dialog_id = this.ulabel.subtasks[current_subtask_key].state.idd_id_front
+        const id_dialog_id: string = current_subtask.state.idd_id;
+        const front_id_dialog_id =
+            this.ulabel.subtasks[current_subtask_key].state.idd_id_front;
 
         // Need the width and inner radius of the pie to re-build it
-        const width: number = this.ulabel.config.outer_diameter
-        const inner_radius = this.ulabel.config.inner_prop * width / 2
+        const width: number = this.ulabel.config.outer_diameter;
+        const inner_radius = (this.ulabel.config.inner_prop * width) / 2;
```

Comment:
Interesting. Thoughts?

Reply:
i guess its slightly more readible? idk order of operations is pretty common sense so imo not necessary